<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInsertDialog</class>
 <widget class="QDialog" name="EmployeeInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>员工编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>员工姓名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>现状:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>来源:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>归口部门:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>推荐人姓名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>身份证号码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学历:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>性别:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>原职业:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入职时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>是否有效:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>家庭住址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeNumberEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeNameEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeStatusEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeSourceEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeUnitEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeRecommandEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeSexEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeEducationEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeePersonalNumberEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeSourceWorkEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeTelEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeAdressEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_EmployeeOkEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="Admin_EmployeeComeDate">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-M-d</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_Commit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_Reset">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_Return">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>520</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="Admin_DecriptionText">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>201</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="Admin_EmployeeBirthdayDate">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-M-d</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>361</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>401</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>label_18</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>Admin_EmployeeNumberEdit</zorder>
  <zorder>Admin_EmployeeNameEdit</zorder>
  <zorder>Admin_EmployeeStatusEdit</zorder>
  <zorder>Admin_EmployeeSourceEdit</zorder>
  <zorder>Admin_EmployeeUnitEdit</zorder>
  <zorder>Admin_EmployeeRecommandEdit</zorder>
  <zorder>Admin_EmployeeSexEdit</zorder>
  <zorder>Admin_EmployeeEducationEdit</zorder>
  <zorder>Admin_EmployeePersonalNumberEdit</zorder>
  <zorder>Admin_EmployeeSourceWorkEdit</zorder>
  <zorder>Admin_EmployeeTelEdit</zorder>
  <zorder>Admin_EmployeeAdressEdit</zorder>
  <zorder>Admin_EmployeeOkEdit</zorder>
  <zorder>Admin_EmployeeComeDate</zorder>
  <zorder>Admin_Commit</zorder>
  <zorder>Admin_Reset</zorder>
  <zorder>Admin_Return</zorder>
  <zorder>Admin_DecriptionText</zorder>
  <zorder>Admin_EmployeeBirthdayDate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
